internal/servers/controller/handlers/credentiallibraries/credentiallibrary_service.go

```diff
@@ -564,16 +569,15 @@ func toStorageVaultLibrary(storeId string, in *pb.CredentialLibrary) (out *vault
 		opts = append(opts, vault.WithDescription(in.GetDescription().GetValue()))
 	}
 
-	attrs := &pb.VaultCredentialLibraryAttributes{}
-	if err := handlers.StructToProto(in.GetAttributes(), attrs); err != nil {
-		return nil, errors.WrapDeprecated(err, op, errors.WithMsg("unable to parse the attributes"))
-	}
+	attrs := in.GetVaultCredentialLibraryAttributes()
 
-	if attrs.GetHttpMethod() != nil {
-		opts = append(opts, vault.WithMethod(vault.Method(strings.ToUpper(attrs.GetHttpMethod().GetValue()))))
-	}
-	if attrs.GetHttpRequestBody() != nil {
-		opts = append(opts, vault.WithRequestBody([]byte(attrs.GetHttpRequestBody().GetValue())))
+	if attrs != nil {
```

Comment:
Technically I think this comparison is actually unnecessary, since using the getters will work fine on nil values.